src/components/views/messages/MessageActionBar.tsx

```diff
@@ -118,8 +120,10 @@ const ReactButton: React.FC<IReactButtonProps> = ({ mxEvent, reactions, onFocusC
     const [menuDisplayed, button, openMenu, closeMenu] = useContextMenu();
     const [onFocus, isActive, ref] = useRovingTabIndex(button);
     useEffect(() => {
+        // when the context menu is opened directly, e.g via mouse click, the onFocus handle is skipped so call manually
+        onFocus();
         onFocusChange(menuDisplayed);
-    }, [onFocusChange, menuDisplayed]);
+    }, [onFocus, onFocusChange, menuDisplayed]);
```

Comment:
TODO: Remove as the strategy changed in https://github.com/matrix-org/matrix-react-sdk/pull/7336#discussion_r768263655